### ISSUE TITLE
Possibility to open an empty directory or file

### ISSUE THREAD

**Issue by yanne** [robotframework/RIDE#237]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=202) by bryan.oakley on 29 Jul 2009*

When told to open an empty directory, RIDE currently exits. That's really
annoying from a usability point of view. Instead, RIDE should have a
special case that creates a valid but empty suite when told to open an
empty directory.

See <a class="closed_ref" href="https://code.google.com/p/robotframework-ride/issues/detail?id=92" title="Cannot open non-empty directory/file unless it contains at least one test case"> issue #92 </a> for discussion. A comment from pekka on that issue suggested
a separate issue should be opened to special-case empty directories in
RIDE. This is that separate issue.

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=202#c3) by @khusa on 5 Oct 2009*


Fixed in <a href="https://code.google.com/p/robotframework-ride/source/detail?r=985">revision 985</a>

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=202#c4) by @pekkaklarck on 5 Oct 2009*


Is this really done also for empty files?

**Comment by yanne** [comment]
> *Originally submitted to [Google Code](http://code.google.com/p/robotframework-ride/issues/detail?id=202#c5) by @khusa on 5 Oct 2009*


Fixed empty file in <a href="https://code.google.com/p/robotframework-ride/source/detail?r=990">revision 990</a>
